### ISSUE TITLE
enum34 dependency causing crash upon install

### ISSUE THREAD

**Issue by schlich** [facultyai/dash-bootstrap-components#429]
I am using Poetry to manage dependencies and upon running `poetry add dash-bootstrap-components` i got [this error](https://github.com/python-poetry/poetry/issues/1122) from the enum34 library.  I fixed it by forcing the enum34 version by manually setting the enum34 version to 1.1.8 as suggested in that thread.

**Comment by tcbegley** [comment]
Hey @schlich 

Thanks for reporting, I was unable to reproduce this with

```
poetry new test
cd test
poetry add dash-bootstrap-components
```

`dash-bootstrap-components` only has one dependency which is `dash` itself (it also has an optional `pandas` dependency for the `Table.from_dataframe` function). In fact, after running those commands, `enum34` was not contained in my `poetry.lock` file, suggesting it's not even an indirect dependency?

My best guess is that `enum34` is a dependency of one of your other dependencies, and `poetry add dash-bootstrap-components` triggered poetry's dependency resolver which resulted in the error.

Which versions of Python are you developing for? Since `enum34` is a backport of a Python 3.4 feature to earlier versions of Python, it's only relevant to those versions, and if you're developing for later versions you might be able to sidestep the whole issue by simply adding something like

```
[tool.poetry.dependencies]
python = "^3.6"
```

in your `pyproject.toml` file?

**Comment by schlich** [comment]
I was already using python 3.8 so not sure where the dependency was either, i didnt see any other packages mentioned in the error message though.  Today I removed enum34 from pyproject.toml and everything went smoothly.  sorry to waste your time and thanks for the help!

**Comment by tcbegley** [comment]
No worries, glad the problem resolved itself.

**Comment by tcbegley** [comment]
Hey @schlich 

I've made some changes to the way we build packages and just made a [prerelease](https://pypi.org/project/dash-bootstrap-components/0.10.6rc1/).

I am now not seeing problems anymore if I set the dash-bootstrap-components version in your `pyproject.toml` to `0.10.6rc1`. If you have a moment would you be able to check if the same is true for you? I'll push out a full release in the next few days.

**Comment by schlich** [comment]
@tcbegley 

ran poetry update with no problems.

**Comment by tcbegley** [comment]
This should now be resolved as of 0.10.6. Feel free to re-open if you encounter more problems.
